config/install/civiremote.settings.yml

```diff
@@ -1,3 +1,4 @@
 cmrf_connector: civiremote
 acquire_civiremote_id: false
 match_blocked_users: false
+match_on_login: false
```

Comment:
On existing installations the initial value should be `true` to keep the current behavior.

Reply:
Current behavior is not to match on login, so no value defaulting to `FALSE` in https://github.com/systopia/civiremote/pull/42/files#diff-4dcbc11d713678e97f70f2321527d825e6e8fc5f6c4f541ee485a050da120f8dR57 should be correct …